training/main/scripts/create_kb.py

```
@@ -131,7 +130,7 @@ def _fetch_wikidata_description(qid: str):
     return item.description.get("da", item.description.get("en", ""))  # type: ignore
 
 
-def _fetch_wikidata_aliases(qid, langs: List[str] = ["da", "en"]):
+def _fetch_wikidata_aliases(qid, langs: list[str] = ["da", "en"]):
```

Ruff (0.8.2)

Missing type annotation for function argument `qid`

(ANN001)